CMakeOptions.cmake

```diff
@@ -136,3 +137,9 @@ if (TCAM_BUILD_GIGETOOL_ONLY)
   set(TCAM_ENABLED_MODULES "tcam-gigetool")
 
 endif (TCAM_BUILD_GIGETOOL_ONLY)
+
+if (TCAM_BUILD_USE_ARM_NEON)
+	set(DUTILS_USE_NEON  ON)
```

Comment:
What is this variable for? 
DUTILS_USE_NEON is not used anywhere else.
So if in doubt just remove it.